nixos/modules/services/web-servers/nginx/default.nix

```
@@ -639,6 +645,22 @@ in
         '';
       };
 
+      mimeTypes = mkOption {
+        type = with types; submodule {
+          freeformType = types.attrsOf (types.listOf str);
+        };
+        default = {};
```

```suggestion
        default = null;
```
I would propose to check against null here.

---

error: A definition for option `services.nginx.mimeTypes' is not of type `attribute set of list of string'.